app/src/main/res/layout/activity_main.xml

```diff
@@ -24,7 +24,7 @@
         android:id="@+id/startStopButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="start"
+        android:text="@string/start"
```

Comment:
adding id's to text - Videos from class

Reply:
Using strings.xml is a good idea, for internationalization.